scripts/cp-editor.js

```diff
@@ -2298,6 +2298,51 @@ H5PEditor.CoursePresentation.prototype.showElementForm = function (element, $wra
             }
           }
         }
+        const isAdvancedText = machineName === 'H5P.AdvancedText';
+        if(isAdvancedText) {
+          if(elementParams.width == this.defaultElementWidthOfContainerInPercent) {
+            const text = elementParams.action.params.text;
+            if(text !== undefined) {
+              const lengthText = text.length;
+              const numberOfParagraphs = text.split('<p>').length - 1;
+              const numberOfListElements = text.split('<li>').length - 1;
+              const elementAspectRatio = this.getDefaultElementAspectRatio('H5P.AdvancedText');
+
+              if(numberOfParagraphs + numberOfListElements == 1 && lengthText < 150) {
+                elementParams.width = this.defaultElementWidthOfContainerInPercent;
+                elementParams.height = (elementParams.width * trueSlideAspectRatio / elementAspectRatio) / 4;
+              }
+              else if(numberOfParagraphs + numberOfListElements == 1 && lengthText >= 150 && lengthText < 300) {
```

Comment:
Could you give a name to these `150` and `300` variables?

Reply:
Of course my lord